### PR TITLE
Help documentation updates

### PR DESCRIPTION
**Pull request by irkaal**
Fixes typo in doc

### CODE REVIEW

File: R/slide.R [hunk 1]
```diff
@@ -117,7 +117,7 @@
 #'   _in the time zone of the index_.
 #'
 #'   This is generally used to define the anchor time to count from,
-#'   which is relevant when the every value is `> 1`.
+#'   which is relevant when every value is `> 1`.
```
Comment (DavisVaughan) [inline R/slide.R:120]: This one was actually almost correct before, but it should have been

```
when the `every` value
```

`every` is a function argument, which is what i was talking about here. Can you please update this to what I showed above?

**Review by DavisVaughan — CHANGES_REQUESTED** [review]
This looks great! We really appreciate these kinds of PRs. I have one comment below, but otherwise this is awesome, thanks!

File: R/slide.R (hunk 1 above)
Comment (irkaal) [reply R/slide.R]: Yes, of course. Sorry about that!

**Comment by DavisVaughan** [comment]
Thank you!

**Pull request merged**